helm_lib/charts/deckhouse_lib_helm/templates/_csi_node.tpl

```diff
@@ -19,8 +19,12 @@ memory: 25Mi
   {{- $serviceAccount := $config.serviceAccount | default "" }}
   {{- $additionalNodeEnvs := $config.additionalNodeEnvs }}
   {{- $additionalNodeArgs := $config.additionalNodeArgs }}
+  {{- $additionalInitContainerEnvs := $config.additionalInitContainerEnvs }}
+  {{- $additionalInitContainerParts := $config.additionalInitContainerParts }}
```

Comment:
we don't use the $config.additionalInitContainerParts in this PR. Let's get rid of this controversial arg.

Reply:
These parameters may come in handy for us in the future, so I added it just in case.